port/aix/omrtime.c

```diff
@@ -83,7 +82,7 @@ omrtime_current_time_nanos(struct OMRPortLibrary *portLibrary, uintptr_t *succes
 	uint64_t nsec = 0;
 	*success = 0;
 	if (0 == clock_gettime(CLOCK_REALTIME, &ts)) {
-		nsec = ((uint64_t)ts.tv_sec * OMRTIME_NANOSECONDS_PER_SECOND) + (uint64_t)ts.tv_nsec;
+		nsec = ((uint64_t)ts.tv_sec * OMRPORT_TIME_DELTA_IN_NANOSECONDS) + (uint64_t)ts.tv_nsec;
```

Comment:
`OMRTIME_NANOSECONDS_PER_SECOND` is defined differently in each `omrtime.c`. Not sure if this change will have compiler-related side-effects in internal builds. `omrtime.c` changes are independent and should be pursued in a separate PR to reduce the risk in a single PR.

Reply:
I created https://github.com/eclipse/omr/issues/7219 for defining `OMRTIME_NANOSECONDS_PER_SECOND` in `omrport.h` and reusing it in `omrtime.c` for the different platforms.